test/test_data.rb

```diff
@@ -848,5 +848,26 @@ def make_three_d_secure(params={})
         :status => 'succeeded',
       }.merge(params)
     end
+
+    def make_apple_pay_domain(params={})
+      {
+        :id => "apwc_test_domain",
+        :object => "apple_pay_domain",
+        :domain_name => "test.com",
+        :livemode => false
```

Comment:
I thought apple pay domain could only be added/retrieve in Live mode? no big deal but might be worth adding a comment